cli/cmd/plugin/ui/web/tanzu-ui/src/shared/types/types.ts

```diff
@@ -27,6 +27,10 @@ export interface VsphereResourceAction extends Action {
     resourceName: string;
 }
 
+export interface AzureResourceAction extends Action {
```

Comment:
Since it's an interface, I would agree with this suggestion. I like keeping the actions themselves named specific to the provider, but this interface should be a common type. Thanks Shimon.

Reply:
Replace `AwsResourceAction`, `VsphereResourceAction`, `AzureResourceAction` with `ResourceAction`